web-ui/src/components/reviews/TeamReviews.jsx

```diff
@@ -224,6 +222,7 @@ const TeamReviews = ({ onBack, periodId }) => {
       members = currentMembers.filter(m => memberIds.includes(m.id));
     }
 
+    console.log('TeamReviews.jsx loadTeamMembers: members.length =', members.length);
```

Comment:
The fact that we get 9 team members when not in approval mode and 8 when in approval mode is the cause for "Michael Kimberlin" being present or not in the displayed review assignments.